src/components/toast/toast.stories.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { withKnobs, text, select } from "@storybook/addon-knobs";
+
+
+import readme from './readme.md';
+
+export default {
+  title: 'Toast',
+  decorators: [withKnobs],
+  parameters: {
+    notes: { markdown: readme },
+  }
+};
+
+export const toastSystem = () => {
+  const variantOptions = { System: 'system', Error: 'error', Success: 'success', Warning: 'warning' };
+  const actionTypeOptions = { Button: 'button', Icon: 'icon' };
+  const buttonActionOptions = { Close: 'close', Custom: 'custom' };
+
+  const showToast = async () => {
+    const newtoast = document.createElement('bds-toast');
```

Comment:
I guess that the pattern you are following is with double quotes, right?

Reply:
actually all the project formats are inconsistent, so we need to fix it with a lint tool soon.